### PR TITLE
Fixed forms related stubs

### LINKED ISSUE #308: Method Latte\Runtime\Filters::breaklines() has invalid return type Latte\Runtime\Html.
in efabrica/phpstan-latte/stubs/Latte/Runtime/Filters.stub:96

### PR DESCRIPTION
**Pull request by MartinMystikJonas**
Resolves #308 

Other changes fixes some errors caused by use of `{control $objectFormControl}`

### CODE REVIEW

File: stubs/Latte/Runtime/Html.php [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<?php
```
Comment (lulco) [inline stubs/Latte/Runtime/Html.php]: this file already exists https://github.com/efabrica-team/phpstan-latte/blob/main/stubs/Latte/Runtime/Html.stub

File: stubs/Nette/Bridges/FormsLatte/Runtime.stub [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+class Runtime {
+
+  /**
+   * @template T of object
+   * @param T|string $item
+   * @param \stdClass $global
+   * @return ($item is object ? T : \Nette\Forms\Control)
+   */
+  public static function item($item, $global): object;
```
Comment (lulco) [inline stubs/Nette/Bridges/FormsLatte/Runtime.stub:13]: this is from latte 3, I'm really interested what we will do if some method will be different in latte 2 and 3 :D

**Review by lulco — APPROVED** [review]

File: stubs/Nette/Bridges/FormsLatte/Runtime.stub (hunk 2 above)
Comment (MartinMystikJonas) [reply stubs/Nette/Bridges/FormsLatte/Runtime.stub:13]: Conditional config withndifferent stubs

File: stubs/Latte/Runtime/Html.php (hunk 1 above)
Comment (MartinMystikJonas) [reply stubs/Latte/Runtime/Html.php]: Weird. I do not have it on local. I must messwd up something with git

**Pull request merged**